cowprotocol/cowswap
# fix(sentry): log error boundary errors to Sentry

# Summary

Logging `ErrorBoundary` errors to Sentry

- Updated sentry to latest
- ~Enabled source maps to improve Sentry error readability~ Disabled as it was breaking the Vercel build :(
- Removed ErrorBoudaryMod file

Sample Sentry error: https://cowprotocol.sentry.io/issues/4415065870/events/a3f15eee22fc43f3a3fbb230ad99e2d7/
![image](https://github.com/cowprotocol/cowswap/assets/43217/ca3f928e-f812-4335-8174-ae43580ec5f7)

# To Test

1. Open limit orders
2. Pick a BUY token
* App WILL CRASH!!!
* Error should be logged in Sentry

# Notes

- ⚠️ DO NOT MERGE BEFORE REVERTING THE COMMIT 39bad8ca3643a35e137d00988dfe5a4a74e52553 ⚠️
- `SENTRY_AUTH_TOKEN` GithubActions envvar might need to be updated
